.github/workflows/lint.yml

```
@@ -32,3 +32,17 @@ jobs:
       run: |
         git diff-index --quiet HEAD -- pwndbg tests
         ./lint.sh
+        
+    - name: Run mypy
+      uses: tsuyoshicho/action-mypy@v4
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        # Change reviewdog reporter if you need [github-pr-check,github-check,github-pr-review].
+        reporter: github-pr-review
```

What happens if you use `github-check` instead?
```suggestion
        reporter: github-check
```
or `github-pr-check`
```suggestion
        reporter: github-pr-check
```